### ISSUE TITLE
command ynh_exec_as absent

### ISSUE THREAD

**Issue by moutonjr** [YunoHost/package_linter#75]
Hi,

Using package_linter with freshly baked OpenProject App 

I got:
```
[Install Script]
! [YEP-2.12] You should not need to use 'sudo', the script is being run as root. (If you need to run a command using a specific user, use 'ynh_exec_as')
```

But there doesn't seem to have such helper:

```
serv:~/git/package_linter$ . /usr/share/yunohost/helpers
serv:~/git/package_linter$ ynh_exec_as
-bash: ynh_exec_as : commande introuvable
```

**Comment by maniackcrudelis** [comment]
Bad communication, this helper is still experimental, you'll find it there:
https://github.com/YunoHost-Apps/Experimental_helpers/blob/master/ynh_exec_as/ynh_exec_as

**Comment by alexAubin** [comment]
Package linter doesn't warn anymore if you "sudo -u someuser"
